service/src/main/java/uk/gov/hmcts/reform/fpl/model/Other.java

```diff
@@ -4,18 +4,23 @@
 import lombok.Builder;
 import lombok.Data;
 
+import java.time.LocalDate;
+
 @Data
 @Builder
 @AllArgsConstructor
 public class Other {
     @SuppressWarnings("membername")
-    private final String DOB;
+    private final LocalDate DOB;        //date not persisting - please investigate
```

Comment:
LocalDateTime ?

Reply:
but it's only a date in the event 😕 String didnt work, Date doesnt work, bit stumped
<img width="253" alt="image" src="https://user-images.githubusercontent.com/15530123/71267078-99e52880-2341-11ea-84bb-5bd56e7f1a75.png">

Reply:
`  {
    "LiveFrom": "01/01/2017",
    "ID": "Others",
    "ListElementCode": "DOB",
    "FieldType": "Date",
    "ElementLabel": "Date of birth",
    "HintText": "For example, 31 3 1980",
    "SecurityClassification": "Public"
  },`

thats the ccd, and for reference we also have
`  {
    "LiveFrom": "01/01/2017",
    "ID": "ChildParty",
    "ListElementCode": "dateOfBirth",
    "FieldType": "Date",
    "ElementLabel": "Date of birth",
    "HintText": "For example, 31 3 2016",
    "SecurityClassification": "Public"
  },`